agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -313,6 +314,17 @@ void main_thread::after_select(bool timeout)
             clear_ready(soc->sta_hal_int_events);
         }
     }
+
+    // Send topology discovery every 60 seconds according to IEEE_Std_1905.1-2013 specification
+    static std::chrono::steady_clock::time_point discovery_timestamp =
```

Comment:
Instead of making this a static local variable I think it should be a member of the main_thread class.

Or, well, actually, I think socket_thread should be refactored to be a proper event loop that can handle timers as well as sockets. But then again, I think all processes should become single threaded... So that's leading us a bit too far :-)